### ISSUE TITLE
crash app

### ISSUE THREAD

**Issue by cong88** [driftyco/cordova-plugin-wkwebview-engine#24]
I had error when open app. Please give me your solution. Thanks!

[4136:1781223] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSArrayM intValue]: unrecognized selector sent to instance 0x17598ce0'
*** First throw call stack:
(0x24ec9b0b 0x24686dff 0x24ecf455 0x24ecd0ab 0x24df7298 0xfd221 0x118e39 0x1047ed 0x10463b 0x2bead5f9 0x2be5d25b 0x2be5ea47 0x2be5e943 0x2be5e8df 0x2bd16b99 0x2be4638f 0x2bce22e3 0x2bce3d07 0x2837442b 0x283748a3 0x24e8b9e7 0x24e8b5d7 0x24e8993f 0x24dd81c9 0x24dd7fbd 0x263f4af9 0x29511435 0xe76b3 0x24a84873)
libc++abi.dylib: terminating with uncaught exception of type NSException

//main.m
#import <UIKit/UIKit.h>

int main(int argc, char* argv[])
{
    @autoreleasepool {
        int retVal = UIApplicationMain(argc, argv, nil, @"AppDelegate");
        return retVal;
    }
}

**Comment by manucorporat** [comment]
@cong88  can you post all the logs? ie. since the app starts

**Comment by cong88** [comment]
This is all logs since the app start.

2016-08-23 09:08:34.968 App[4276:1853475] Apache Cordova native platform version 4.1.1 is starting.
2016-08-23 09:08:34.973 App[4276:1853475] Multi-tasking -> Device: YES, App: YES
2016-08-23 09:08:35.098 App[4276:1853475] CDVWKWebViewEngine: trying to inject XHR polyfill
2016-08-23 09:08:35.129 App[4276:1853475] CDVWKWebViewEngine will reload WKWebView if required on resume
2016-08-23 09:08:35.179 App[4276:1853475] Using WKWebView
2016-08-23 09:08:35.200 App[4276:1853475] [CDVTimer][handleopenurl] 0.317991ms
2016-08-23 09:08:35.209 App[4276:1853475] [CDVTimer][intentandnavigationfilter] 5.959988ms
2016-08-23 09:08:35.210 App[4276:1853475] [CDVTimer][gesturehandler] 0.231028ms
2016-08-23 09:08:35.238 App[4276:1853475] [CDVTimer][splashscreen] 27.797043ms
2016-08-23 09:08:35.239 App[4276:1853475] Starting Facebook Connect plugin
2016-08-23 09:08:35.239 App[4276:1853475] [CDVTimer][facebookconnectplugin] 0.455976ms
2016-08-23 09:08:35.239 App[4276:1853475] [CDVTimer][googleplus] 0.198007ms
2016-08-23 09:08:35.261 App[4276:1853475] [CDVTimer][statusbar] 21.731973ms
2016-08-23 09:08:35.262 App[4276:1853475] DisableShakeToEdit plugin initialized.
2016-08-23 09:08:35.262 App[4276:1853475] [CDVTimer][disableshaketoedit] 0.428975ms
2016-08-23 09:08:35.262 App[4276:1853475] [CDVTimer][TotalPluginStartup] 63.079000ms
2016-08-23 09:08:35.277 App[4276:1853475] createNotificationChecker
2016-08-23 09:08:35.278 App[4276:1853475] not coldstart
2016-08-23 09:08:35.297 App[4276:1853475] active
2016-08-23 09:08:35.298 App[4276:1853475] PushPlugin skip clear badge
2016-08-23 09:08:39.902 App[4276:1853514] CDVWKWebViewEngine: XHR intercepted: build/app.html
2016-08-23 09:08:40.576 App[4276:1853515] CDVWKWebViewEngine: XHR intercepted: build/pages/home/home.html
2016-08-23 09:08:40.597 App[4276:1853505] CDVWKWebViewEngine: XHR intercepted: build/components/header/header.html
2016-08-23 09:08:40.609 App[4276:1853505] CDVWKWebViewEngine: XHR intercepted: build/components/salons/salons.html
2016-08-23 09:08:40.615 App[4276:1853515] Push Plugin register called
2016-08-23 09:08:40.615 App[4276:1853515] PushPlugin.register: setting badge to true
2016-08-23 09:08:40.615 App[4276:1853515] PushPlugin.register: clear badge is set to 1
2016-08-23 09:08:40.615 App[4276:1853515] PushPlugin.register: better button setup
2016-08-23 09:08:40.616 App[4276:1853515] GCM Sender ID (null)
2016-08-23 09:08:40.617 App[4276:1853515] Using APNS Notification
2016-08-23 09:08:40.677 App[4276:1853475] Push Plugin register success: <4cd01c10 e383c044 615e54c7 b16fd98c 54e48b48 bacca5c0 e109f671 7d6b0bb3>
2016-08-23 09:08:43.760 App[4276:1853475] -[__NSArrayM intValue]: unrecognized selector sent to instance 0x14d98080
2016-08-23 09:08:43.761 App[4276:1853475] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSArrayM intValue]: unrecognized selector sent to instance 0x14d98080'
*** First throw call stack:
(0x24ec9b0b 0x24686dff 0x24ecf455 0x24ecd0ab 0x24df7298 0x2a621 0x52811 0x3e1c5 0x3e013 0x2bead5f9 0x2be5d25b 0x2be5ea47 0x2be5e943 0x2be5e8df 0x2bd16b99 0x2be4638f 0x2bce22e3 0x2bce3d07 0x2837442b 0x283748a3 0x24e8b9e7 0x24e8b5d7 0x24e8993f 0x24dd81c9 0x24dd7fbd 0x263f4af9 0x29511435 0x14ab3 0x24a84873)
libc++abi.dylib: terminating with uncaught exception of type NSException
(lldb)

**Comment by ammichael** [comment]
I'm having crash issues too. The app loads fine the first time, but when I minimize and reopen it, it immediately crashes.

Here the log:

```
2016-08-22 23:19:27.189865 Finances[4420:1051355] CDVWKWebViewEngine reloadIfRequired
2016-08-22 23:19:27.189991 Finances[4420:1051355] CDVWKWebViewEngine reloadIfRequired WKWebView.title:
2016-08-22 23:19:27.190020 Finances[4420:1051355] CDVWKWebViewEngine reloadIfRequired reload: 1
2016-08-22 23:19:27.190042 Finances[4420:1051355] CDVWKWebViewEngine reloading!
2016-08-22 23:19:27.574134 Finances[4420:1051404] CDVWKWebViewEngine: XHR intercepted: templates/finances.html
2016-08-22 23:19:27.774204 Finances[4420:1051355] *** Terminating app due to uncaught exception 'NSRangeException', reason: 'Cannot remove an observer <CDVSplashScreen 0x17009a5e0> for the key path "frame" from <UIView 0x105d0e5d0> because it is not registered as an observer.'
*** First throw call stack:
(0x18f7a01c0 0x18e1d855c 0x18f7a0108 0x1901afb78 0x1901af654 0x10009286c 0x1000942cc 0x100091c94 0x1001b273c 0x10019c2b4 0x10019c114 0x198c15888 0x198bb6810 0x198bb8b54 0x198a11a68 0x198b92508 0x1989dacdc 0x1989dd2f4 0x193e9d874 0x193e9daa0 0x18f74e278 0x18f74dbc0 0x18f74b7c0 0x18f67a048 0x1910fb198 0x1955e4bd0 0x1955df908 0x100090c9c 0x18e65c5b8)
libc++abi.dylib: terminating with uncaught exception of type NSException
(lldb) 

 [self.viewController.view removeObserver:self forKeyPath:@"frame"];
```

**Comment by manucorporat** [comment]
@ammichael can you try to remove plugins/ and platforms/ folder recreate all the cordova files from scratch?

if the problem persist, can you provide a minimum repo (or your whole app, I don't care) that reproduces the issue? so I can try to fix it

**Comment by manucorporat** [comment]
@jgw96 I think we can close this issue, right? it's been 27 days ago since last contact.

**Comment by ammichael** [comment]
@manucorporat Hey there. I haven't had time to work in the project that was crashing since then. Well, at the time I've reseted ionic state, so it wasn't the solution.

I just figured out that the problem was having the title changing (so the whole app reloaded whenever I resumed from background), AND, while it was reloading, it was trying to access splaschscreen plugin to show/hide, resulting in crash.

I just fixed the title tag and everything is fine now.

Thanks a lot!
